src/ol/webgl/ShaderBuilder.js

```diff
@@ -580,6 +574,7 @@ void main(void) {
   float s = sin(v_angle);
   coordsPx = vec2(c * coordsPx.x - s * coordsPx.y, s * coordsPx.x + c * coordsPx.y);
   gl_FragColor = ${this.symbolColorExpression_};
+  gl_FragColor.rgb *= gl_FragColor.a;
```

Comment:
BTW, currently `u_globalAlpha` is not applied here. Is this OK for a symbol...?

Reply:
Oh, right, maybe that was missing. I think this should be done in another PR, I have the feeling this isn't tested anywhere